oak_functions_launcher/src/lookup.rs

```diff
@@ -16,42 +16,98 @@
 
 use crate::{
     channel::ConnectorHandle,
-    schema::{self, OakFunctionsAsyncClient},
+    schema::{self, LookupDataChunk, OakFunctionsAsyncClient, UpdateAction, UpdateStatus},
 };
 use anyhow::{anyhow, Context};
 use hashbrown::HashMap;
 use prost::Message;
-use std::{fs, path::PathBuf};
+use std::{fs, path::PathBuf, vec::IntoIter};
 use ubyte::ByteUnit;
 
+struct UpdateClient<'a> {
+    inner: &'a mut OakFunctionsAsyncClient<ConnectorHandle>,
+}
+
+impl<'a> UpdateClient<'_> {
```

Comment:
do you have to give a name to the first lifetime parameter here? I don't think you're explicitly referencit it anywhere.

Reply:
No, I don't, thanks!